Lombiq.Tests.UI/OrchardCoreUITestBase.cs

```diff
@@ -13,49 +13,40 @@ namespace Lombiq.Tests.UI;
 
 public abstract class OrchardCoreUITestBase
 {
+    private const string AppFolder = "AppFolder";
+
     private static readonly object _snapshotCopyLock = new();
 
     protected readonly ITestOutputHelper _testOutputHelper;
 
     private static bool _appFolderCreated;
 
+    protected abstract string AppAssemblyPath { get; }
+
     protected virtual Size StandardBrowserSize => CommonDisplayResolutions.Standard;
     protected virtual Size MobileBrowserSize => CommonDisplayResolutions.NhdPortrait;
 
-    protected abstract string AppAssemblyPath { get; }
+    static OrchardCoreUITestBase() => AtataFactory.SetupShellCliCommandFactory();
```

Comment:
No, this will break customizability. Pasting what I wrote [here](https://github.com/Lombiq/UI-Testing-Toolbox/pull/119#discussion_r874210773):

> We can't. Static constructors are called when the type is first referenced. So if we want to let the library consumer customize it in the `OrchardCoreUITestBase` overload as described in _Linux.md_, then we must be absolutely certain that this code has run before the setup customization. In general you should only use static constructor to set data that will be immediately interacted with. Every test calls to `OrchardCoreUITestBase` very early so that's perfect.
In this case I have verified that `AtataFactory`'s static constructor actually executes later than e.g. the configuration in `Lombiq.OSOCE.Tests.UI.UITestBase.ExecuteTestAsync`.

Reply:
I (R#) just moved this line from the bottom of the file up here.

Looks to me like this is what we have after you did [this](https://github.com/Lombiq/UI-Testing-Toolbox/pull/119#discussion_r874224878):
> encapsulate this into AtataFactory so we don't have basic Atata configuration in multiple places

Shouldn't that be good now?